pgx/v1.py

```diff
@@ -295,7 +295,9 @@ def available_games() -> Tuple[EnvId, ...]:
     return get_args(EnvId)
 
 
-def make(env_id: EnvId):  # noqa: C901
+def make(  # noqa: C901
+    env_id: EnvId, *, dds_hash_table_path: Optional[str] = None
```

Comment:
```suggestion
    env_id: EnvId
```

この変更はダメですね。他のゲームみてもわかると思うんですけど、
Goとかもパラメータありますけど、ここではデファクトのものしか呼び出せないようにしてます。
（パラメータで調整できないようになっています）。

これは消してください。

Reply:
ブリッジを真面目にやりたい人は `make` ではなく、 `BridgeBidding` を直接呼び出す想定でいいと思います。

あとは、 セッターメソッドでテーブルを上書きできるようにしておいたらいいと思います。

Reply:
なのでテスト内でも `make` 使わずに `BridgeBidding` 直接呼び出したらいいと思います。